### PR TITLE
dns/ddclient Add support for ddclient 3.10.0 dns provider

### PR DESCRIPTION
**Pull request by lucaschoeneberg**
Add 1984, ClouDNS, Dinahosting, DNSExit, DonDominio, Freemyip, godaddy, Hetzner, Key-Systems, NearlyFreeSpeech.NET, Njal.la(thanks to satrapes), sitelutions, woima and Yandex for ddclient 3.10.0.

I hope that is okay that I have taken the changes from https://github.com/opnsense/plugins/pull/3129.

### CODE REVIEW

**Comment by trunet** [comment]
ddclient [3.10.0](https://github.com/ddclient/ddclient/releases/tag/v3.10.0) had the first stable release after 3 years on Oct 10th.

this PR should fix at least #3143 #3135

Please, give some love and have this rolling...

File: dns/ddclient/pkg-descr [hunk 1]
```diff
@@ -5,6 +5,23 @@ WWW: https://github.com/ddclient/ddclient
 
 Plugin Changelog
 ================
+2.0
```
Comment (fichtner) [inline dns/ddclient/pkg-descr:8]:
```suggestion

2.0
```

File: dns/ddclient/pkg-descr [hunk 2]
```diff
@@ -6,6 +6,24 @@ WWW: https://github.com/ddclient/ddclient
 Plugin Changelog
 ================
 
+2.0
```
Comment (fichtner) [inline dns/ddclient/pkg-descr:9]:
```suggestion
1.10
```

**Comment by fichtner** [comment]
Merged, thanks!

**Comment by dabruh** [comment]
Nice! Any idea when this package will be installable on my OPNSense box? :-)

**Comment by fichtner** [comment]
The development version has it installable… os-ddclient-devel

**Comment by dabruh** [comment]
Sorry, I have no clue where from to install that. I tried setting Type: Development in /ui/core/firmware#settings but it didn't display any os-ddclient-devel package. I've got OPNSense 22.7.9.

When can I expect this package to find its way to the community branch?

**Comment by fichtner** [comment]
No ETA yet.

**Comment by Xyrodileas** [comment]
Will this be available in the business branch as well ?

**Pull request merged**